cns/5.0/rn/release-information/release-notes-5-2-1.adoc

```diff
@@ -0,0 +1,45 @@
+== 5.2.1 Release Notes
+
+//'''
+//
+//title: 5.2.1
+//type: list
+//url: "/5.0/release-notes/5.2.1/"
+//menu:
+//  5.0:
+//    parent: "release-notes"
+//    identifier: 5.2.1
+//    weight: 87
+//
+//'''
+
+April 12, 2022
+
+=== Release information
+
+Starting with this release, Enforcer client versions will be backwards compatible for a 9-month period.
+
+=== New Features
+
+* Support for IPv6 OAM ping
+* Flow reporting interval changes
+* Support for Enforcer & Processing Unit alerts
+* Enforcer lookup improvements - ruleset v2 only feature
+* Long-lived connections
+* Syslog capabilities to enforcer
+* API server support on Kubernetes
+* Enforcer support for non-transparent proxies
+
+=== Resolved Issues
+
+* *CNS-4117*: Auto-scale issues seen with Squall service are addressed.
+* *CNS-4717*: SSL errors seen on UI when requests hit rate limits are addressed.
+* *CNS-3907*: Addressed the slowness with tags retrieval with a large number of Enforcers.
+* *CNS-3835*: Prevent TUF repo error logging on Chocobo service pods.
```

Comment:
with this fix, are we preventing TUF repo error logging on Chocobo service pods?